protocol-units/da-sequencer/node/src/error.rs

```diff
@@ -23,6 +23,8 @@ pub enum DaSequencerError {
 	DeserializationFailure,
 	#[error("Signature was invalid")]
 	InvalidSignature,
+	#[error("Non-whitelisted signer")]
+	InvalidSigner,
```

Comment:
Invalid suggests format or other errors, whereas this case explicitly deals with authorization. 
```suggestion
	UnauthorizedSigner,
```